blocks/v2-footer/v2-footer.css

```diff
@@ -0,0 +1,590 @@
+    display: flex;
+    width: 100%;
+    padding: 24px;
+    justify-content: flex-start;
+    align-items: baseline;
+    background-color: var(--c-secondary-graphite);
+  }
+
+  .v2-footer .v2-pre-footer-list-wrapper h3 {
+    color: var(--primary-white-ffffff, #FFF);
+    font-family: var(--ff-body);
+    font-size: 12px;
+    line-height: 16px; /* 133.333% */
+    letter-spacing: 1.92px;
+    text-transform: uppercase;
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+  }
+
+  /* stylelint-disable-next-line no-descending-specificity */
+  .v2-footer .v2-pre-footer-list-wrapper h3 .icon-dropdown-caret svg{ 
+    height: 20px;
+    width: 20px;
+  }
+
+  .v2-pre-footer-list .icon-dropdown-caret {
+    margin-left: auto;
+    transition: transform 0.5s;
+    transform: rotate(90deg);
+  }
+  
+  .v2-pre-footer-list.expand .icon-dropdown-caret {
+    transform: rotate(-90deg);
+  }
+
+  .v2-footer .v2-pre-footer-list-wrapper ul{
+    width: 100%;
+    gap: 10px;
+    flex-flow: column;
+  }
+
+  .v2-pre-footer-list{
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .v2-pre-footer-list-item {
+    transition: 300ms max-height ease;
+    will-change: max-height;
+    overflow: hidden;
+    max-height: 0;
+  }
+
+  /* stylelint-disable-next-line no-descending-specificity */
+  .v2-footer .v2-pre-footer-list-item a {
+    display: flex;
+    padding-left: 16px;
+    padding-right: 16px;
+    padding-bottom: 16px;
+  }
+
+  /* desktop   */
+  @media (min-width: 1200px) {
+    .v2-footer .v2-footer-content {
+      display: flex;
+      flex-flow: row wrap;
+      padding-top: 96px;
+    }
+  
+    .v2-footer .v2-footer-links-section {
+      display: flex;
+      justify-content: space-between;
+      align-items: flex-start;
+      gap: 20px;
+      flex-wrap: nowrap;
+    }
+
+    .v2-footer .v2-footer-links-section ul {
+      min-width: 0;
+      width:160px;
+    }
+  
+    .v2-footer .v2-footer-links-col {
+      width: 185px;
+    }
+  
+    /* stylelint-disable-next-line no-descending-specificity */
+    .v2-footer .v2-footer-bottom-section li {
+      padding: 10px;
+      width: auto;
+    }
+  
+    .v2-footer .v2-footer-links-col-1 ul li {
+      width: 100%;
+    }
+  
+    .v2-footer .v2-footer-bottom-section {
+      width: 100%;
+      margin-top: 65px;
+    }
+  
+    .v2-footer .v2-footer-bottom-section ul {
+      width: 100%;
+      display: flex;
+      flex-wrap: nowrap;
+      max-width: unset;
+      align-items: flex-start;
+      justify-content: left;
+      margin-top: 22px;
+    }
+
+    /* stylelint-disable-next-line no-duplicate-selectors */
+    .v2-footer .v2-footer-main-content{
+      display: grid; 
+      grid-template-columns: 1fr 1fr 1fr; 
+      grid-template-rows: 1fr 1fr; 
+      gap: 0 0; 
+      /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+      grid-template-areas: 
+      "v2-logo v2-content v2-form"
```

Comment:
Use single quotes and be careful with the indentation in all the document.

Reply:
Updated